package/com.unity.formats.usd/Tests/USD.NET.Unity/InstancingTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Linq;
+using NUnit.Framework;
+using pxr;
+using UnityEngine;
+
+namespace USD.NET.Unity.Tests
+{
+    class InstancingTests : UsdTests
+    {
+        Scene CreatePointInstancerScene()
+        {
+            var tmpScenePath = CreateTmpUsdFile("PointInstancerTest.usda");
```

Comment:
I would make the file name be random, just to make sure you never fall on the same file twice.